cmd/geth/main.go

```diff
@@ -153,6 +153,7 @@ var (
 		utils.MinerNotifyFullFlag,
 		utils.IgnoreLegacyReceiptsFlag,
 		configFileFlag,
+		utils.ExportReceiptsFlag,
```

Comment:
I don't think you need to add this flag to the geth binary. It should be sufficient to add it to the export command